### DISCUSSION TITLE
Regarding internship credits and Mentor details

### DISCUSSION THREAD

**Discussion by srujan-rai-k** [github/india#808] [Students]
Hello mentors, I have completed all the modules including the project and received the final internship certificate. I have 2 questions
1. Is this certificate eligible to get internship credits from my college ? 
2. If eligible, our college has asked us to provide with mentors details like name, contact no, email id. Could you please provide these details ? 

Thank you. 
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-mentor4

**Comment by ey-mentor1** [comment]
Yes you can submit this for the college credits.
Regarding second query, please send the mail to support team, the official names for the project would be shared by them only.
